.github/workflows/build.yml

```diff
@@ -48,78 +48,15 @@ jobs:
-        run: cd SampleApps/SPMTest && swift package resolve
-      - name: Build & archive SPMTest
-        run: xcodebuild -project 'SampleApps/SPMTest/SPMTest.xcodeproj' -scheme 'SPMTest' clean build archive CODE_SIGNING_ALLOWED=NO
-  cocoapods_xcode_13:
-    name: CocoaPods (Xcode 13)
-    runs-on: macOS-11
-    steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
-        with:
-           ref: ${{ github.event.pull_request.head.ref }}
-           repository: ${{ github.event.pull_request.head.repo.full_name }}
-      - name: Use Xcode 13
-        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
-      - name: Install CocoaPod dependencies
-        run: pod install
-      - name: Run pod lib lint 
-        run: pod lib lint
-  carthage_xcode_13:
-    name: Carthage (Xcode 13)
-    runs-on: macOS-11
-    steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-      - name: Use Xcode 13
-        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
-      - name: Remove SPMTest
-        run: |
-          git checkout $GITHUB_HEAD_REF
-          rm -rf SampleApps/SPMTest
-          rm -rf /Users/runner/Library/Developer/Xcode/DerivedData
-          git add SampleApps/SPMTest
-          git commit -m 'Remove SPMTest app to avoid Carthage timeout'
-      - name: Use current branch
-        run: echo 'git "file://'$PWD'" "'$GITHUB_HEAD_REF'"' > SampleApps/CarthageTest/Cartfile
-      - name: Run carthage build
-        run: carthage build --no-skip-current --use-xcframeworks
-      - name: Run carthage update
-        run: cd SampleApps/CarthageTest && carthage update --use-xcframeworks
-      - name: Build CarthageTest
-        run: xcodebuild -project 'SampleApps/CarthageTest/CarthageTest.xcodeproj' -scheme 'CarthageTest' clean build CODE_SIGNING_ALLOWED=NO
-      - name: Run carthage build without --use-xcframeworks
-        run: |
-          ./carthage.sh build --no-skip-current
-          ./carthage.sh archive "BraintreeAmericanExpress" "BraintreeApplePay" "BraintreeCard" "BraintreeCore" "BraintreeDataCollector" "BraintreePaymentFlow" "BraintreePayPal" "BraintreeSEPADirectDebit" "BraintreeThreeDSecure" "BraintreeUnionPay" "BraintreeVenmo" "PayPalDataCollector" "BraintreePayPalNativeCheckout" --output Braintree.framework.zip
-  spm_xcode_13:
-    name: SPM (Xcode 13)
-    runs-on: macOS-11
-    steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
-        with:
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-      - name: Use Xcode 13
-        run: sudo xcode-select -switch /Applications/Xcode_13.0.app
+        run: sudo xcode-select -switch /Applications/Xcode_14.1.app
```

Comment:
just a question, why do we delete Carthage and Cocoapods from the build process?

Reply:
just trying to understand, because cocoapods and carthage are run in diferent jobs. Shouldnt we keep on running those jobs but with Xcode 14.1? For example, for cocoapods, we are not running pod install and pod lib lint anymore.

Reply:
I think it's the diff that's confusing. the cocoapods step is still happening on this PR : 
https://github.com/braintree/braintree_ios/pull/1018/files/5ea6893111355be78fe06af9fd0fd9d346201b90#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R8-R21

same with carthage: 
https://github.com/braintree/braintree_ios/pull/1018/files/5ea6893111355be78fe06af9fd0fd9d346201b90#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R22-R48

Reply:
got it. I didnt realize we were running both 13 and 14 before this PR. Now it makes sense. Thank you!

Reply:
Yeh, the diff is weird on this. But we're keeping CocoaPods & Carthage jobs, just only for 1 version of Xcode now.